java/test/jmri/jmrit/operations/PackageTest.java

```diff
@@ -16,7 +16,7 @@
         XmlLoadTest.class, // no tests in class itself
         BundleTest.class, 
         CommonConductorYardmasterPanelTest.class, 
-        jmri.jmrit.operations.locations.PackageTest.class, // fixed references to Swing, 10/10/2012
+//        jmri.jmrit.operations.locations.PackageTest.class, // fixed references to Swing, 10/10/2012
```

Comment:
Why are you disabling all tests in the jmri.jmrit.operations.locations package when run within Ant?

Reply:
@rhwood, this is actually a duplicate we somehow introduced.  The line actually should be removed.